packages/compiler/core/cli/cli.ts

```diff
@@ -103,6 +103,11 @@ async function main() {
             string: true,
             describe: "List of areas that should have the trace shown. e.g. `import-resolution.*`",
           })
+          .option("config", {
+            type: "string",
+            describe:
+              "The path to a TypeSpec config YAML file. Defaults to {cwd}/tspconfig.yaml. If a folder is specified, the CLI will attempt to search for the file by moving up the folder hierarchy.",
```

Comment:
Is it more accurate to say that this defaults to {cwd}. Because if I set this to {cwd}/banana.yaml, it won't go up and look at {cwd}/../banana.yaml, right?

Reply:
Good point. As written it will only do the recursive search if you give it a folder. If you give it a specific path, it's either there or not.

Reply:
Fixed.